lib/xgu/xgu.h

```diff
@@ -36,6 +36,17 @@ typedef enum {
     XGU_POLYGON = NV097_SET_BEGIN_END_OP_POLYGON
 } XguPrimitiveType;
 
+typedef enum {
+    XGU_FUNC_NEVER = 0x200,
+    XGU_FUNC_LESS,
+    XGU_FUNC_EQUAL,
+    XGU_FUNC_LESS_OR_EQUAL,
+    XGU_FUNC_GREATER,
+    XGU_FUNC_NOT_EQUAL,
+    XGU_FUNC_GREATER_OR_EQUAL,
+    XGU_FUNC_ALWAYS
```

Comment:
Please set these values explicitly. We use the enums to make user code safer, not to implicitly get values for constants (as the hardware is very well defined, so it's good to explicitly have these values in the source code, just in case someone stupid adds another value later, or messes up the order)